### PR TITLE
Updated models for revscoring-2.8.2

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: model_info/eswikiquote.damaging.md [hunk 1]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
-	counts (n=11732):
-		label        n         ~True    ~False
-		-------  -----  ---  -------  --------
-		True      1019  -->      827       192
-		False    10713  -->     1003      9710
+	counts (n=9758):
+		label       n         ~True    ~False
+		-------  ----  ---  -------  --------
+		True      849  -->      687       162
+		False    8909  -->      750      8159
```
Comment (halfak) [inline model_info/eswikiquote.damaging.md:25]: Big change in the # of observations.  Not sure why.

Comment (halfak) [reply model_info/eswikiquote.damaging.md:25]: Confirmed that there are a ton of revisions that have been deleted.

Comment (halfak) [reply model_info/eswikiquote.damaging.md:25]: Here's an example of a missing revision: https://es.wikiquote.org/wiki/?diff=326324

There's large blocks of them that are deleted.  Here's some output from a diff: 

< 326324
< 326325
< 326326
< 326327
< 326328
84,98d78
< 326330
< 326331
< 326332
< 326333
< 326334
< 326335
< 326336
< 326337
< 326338
< 326339
< 326340
< 326341
< 326342
< 326343
< 326344

I think we should proceed.

Comment (halfak) [reply model_info/eswikiquote.damaging.md:25]: See https://phabricator.wikimedia.org/T254785

**Comment by halfak** [comment]
I've scanned all of the model_info.  It looks like the only significant change is the one I noted above.

**Comment by halfak** [comment]
When I look at this, it seems like the build is still in progress, but when I click "details" it says it completed successfully.

**Comment by HAKSOAT** [comment]
LGTM

**Pull request merged**